docs/_include/variable_hydro.rst

```diff
@@ -63,7 +63,16 @@ every body in a simulation. To implement variable hydrodynamics for a given body
         ``body(1) = bodyClass({'H5FILE_1.h5','H5FILE_2.h5','H5FILE_3.h5','H5FILE_4.h5','H5FILE_5.h5');``
 
     If only one H5 file is used to initialize a body object, variable hydrodynamics
-    will not be used, regardless of the ``option`` flag below.
+    will not be used, regardless of the ``option`` flag below. 
+
+    For variable mass cases, a vector can also be used for the variable hydrodynamics mass, 
+    inertia, and inertia products:
+
+        ``body(1).variableHydro.mass = [mass1, mass2, mass3, mass4, mass5];``
+
+    If no mass vector is specified but ``body(1).mass`` is set to equilibrium, the data from H5 
+    files will be used to calculate the variable mass based on displaced volume and water density. 
+    However, the inertia vector will still need to be specified or else it will be assumed constant.
```

Comment:
In the equilibrium case doesn't `inertia` have to be defined by the user? Otherwise it will be empty and the simulation won't run
```suggestion
    However, the inertia vector will still need to be specified.